kakao/src/main/kotlin/com/agoda/kakao/edit/KTextInputLayout.kt

```diff
@@ -20,21 +21,21 @@ class KTextInputLayout : KBaseView<KTextInputLayout>, TextInputLayoutAssertions
     constructor(function: ViewBuilder.() -> Unit) : super(function) {
         edit = KEditText {
             isDescendantOfA(function)
-            isInstanceOf(TextInputEditText::class.java)
+            withMatcher(ViewMatchers.isAssignableFrom(EditText::class.java))
```

Comment:
Can you please extract this matcher and make it available inside `ViewBuilder`?

Reply:
done